mod_ci/controllers.py

```diff
@@ -622,19 +621,27 @@ def start_ci():
 
 
 def update_build_badge(status, test):
+    """
+    Build status badge for current test to be displayed on sample-platform.ccextractor.org
+
+    :param status: current testing status
+    :type status: str
+    :param test: current commit that is tested
+    :type test: Test
+    :return: Nothing.
+    :rtype: None
+    """
     if test.test_type == TestType.commit:
+        test.status = Status()
```

Comment:
Why are you doing this? This makes still no sense. You pass in a `status` as an argument to this method, and then you're not using it?

Reply:
change ```test.status``` into ```status``` ?

Reply:
oooh
i understand now about your comment above

i misread ah

Reply:
changed

thanks :)